src/Cache/CachedTrait.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Cache;
+
+use Psr\Cache\CacheException;
+use Psr\Cache\CacheItemPoolInterface;
+
+/**
+ * @internal
+ */
+trait CachedTrait
+{
+    /** @var CacheItemPoolInterface */
+    private $cacheItemPool;
+    private $localCache = [];
+
+    private function getCached(string $cacheKey, callable $getValue)
+    {
+        if (isset($this->localCache[$cacheKey])) {
```

Comment:
Actually, I think this is a bug. We should support storing `null` values in the cache.